cli/internal/context/context.go

```diff
@@ -103,6 +105,30 @@ func WithGraph(rootpath string, config *config.Config) Option {
 		}
 		c.RootPackageJSON = pkg
 
+		// If turbo.json exists, we use that
+		// If pkg.Turbo exists, we warn about running the migration
+		// Use pkg.Turbo if turbo.json doesn't exist
+		// If neither exists, it's a fatal error
+		if !fs.FileExists("turbo.json") {
```

Comment:
**suggestion:** is `rootpath` available? i think we should make this check aware of it in case the `--cwd` flag is passed. So likely `fs.FileExists(filepath.Path(rootpath, "turbo.json"))` or something

Reply:
Done